lib/katello/tasks/test.rake

```diff
@@ -39,8 +49,82 @@ namespace :test do
 
       Rake::Task[test_task.name].invoke
     end
-  end
 
+    namespace :test do
+
+      desc "Run the Katello plugin unit models test suite."
+      task :models => ['db:test:prepare'] do
+
+        test_task = Rake::TestTask.new('katello_models_test_task') do |t|
+          t.libs << ["test", "#{Katello::Engine.root}/test"]
+          t.test_files = [
+            "#{Katello::Engine.root}/test/models/**/*_test.rb",
+          ]
+          t.verbose = true
+        end
+
+        Rake::Task[test_task.name].invoke
+      end
+
+      desc "Run the Katello plugin unit actions test suite."
+      task :actions => ['db:test:prepare'] do
+
+        test_task = Rake::TestTask.new('katello_actions_test_task') do |t|
+          t.libs << ["test", "#{Katello::Engine.root}/test"]
+          t.test_files = [
+            "#{Katello::Engine.root}/test/actions/**/*_test.rb",
+          ]
+          t.verbose = true
+        end
+
+        Rake::Task[test_task.name].invoke
+      end
+
+      desc "Run the Katello plugin unit controllers test suite."
+      task :controllers => ['db:test:prepare'] do
+
+        test_task = Rake::TestTask.new('katello_controllers_test_task') do |t|
+          t.libs << ["test", "#{Katello::Engine.root}/test"]
+          t.test_files = [
+            "#{Katello::Engine.root}/test/controllers/**/*_test.rb",
+          ]
+          t.verbose = true
+        end
+
+        Rake::Task[test_task.name].invoke
+      end
+
+      desc "Run the Katello plugin unit glue test suite."
+      task :glue => ['db:test:prepare'] do
+
+        test_task = Rake::TestTask.new('katello_glue_test_task') do |t|
+          t.libs << ["test", "#{Katello::Engine.root}/test"]
+          t.test_files = [
+            "#{Katello::Engine.root}/test/glue/**/*_test.rb",
+          ]
+          t.verbose = true
+        end
+
+        Rake::Task[test_task.name].invoke
+      end
+
+      desc "Run the Katello plugin unit lib test suite."
+      task :lib => ['db:test:prepare'] do
+
+        test_task = Rake::TestTask.new('katello_lib_test_task') do |t|
+          t.libs << ["test", "#{Katello::Engine.root}/test"]
+          t.test_files = [
+            "#{Katello::Engine.root}/test/lib/**/*_test.rb",
+          ]
+          t.verbose = true
+        end
+
+        Rake::Task[test_task.name].invoke
+      end
+
+    end
+  end
+    
```

Comment:
@ehelms I added these rake tasks for individual test folders we have been using for rails 4. They have proved to be useful and I figured others could benefit from using them, let me know if you want them removed.